cmd/ci-operator-config-mirror/main.go

```diff
@@ -35,6 +38,8 @@ func gatherOptions() options {
 	fs.StringVar(&o.configDir, "config-path", "", "Path to directory containing ci-operator configurations")
 	fs.StringVar(&o.toOrg, "to-org", "", "Name of the organization which the ci-operator configuration files will be mirrored")
 
+	fs.BoolVar(&o.deprecate, "deprecate-configs", true, "Deletes the existing but deprecated configuration")
```

Comment:
I'd like to see an explanation about what a "deprecated" configuration means. "Delete all ci-operator configuration that exist for repos in the target org, but which would not be created by the current ci-operator-config-mirror run"